internal/daemon/daemon.go

```diff
@@ -195,7 +199,11 @@ func (d *Daemon) init(listenPort string, schemaExtensions []schema.Update, apiEx
 		resources.InternalEndpoints,
 		resources.PublicEndpoints,
 	}
-	serverEndpoints = append(serverEndpoints, coreEndpoints...)
+
+	for _, server := range d.extensionServers {
+		serverEndpoints = append(serverEndpoints, server.Resources...)
+	}
+
 	ctlServer := d.initServer(serverEndpoints...)
 	ctl := endpoints.NewSocket(d.shutdownCtx, ctlServer, d.os.ControlSocket(), d.os.SocketGroup)
```

Comment:
What do you think about having something similar like `addCoreServers` also for the Unix socket server? Or should it be part of `addCoreServers`?